chains/1.json

```diff
@@ -30,5 +30,85 @@
     "symbol": "T",
     "decimals": 18,
     "logoURI": "../images/threshold.svg"
-  }
+  },
+  {
+    "address": "0x4e3fbd56cd56c3e72c1403e103b45db9da5b9d2b",
+    "chainId": 1,
+    "name": "Convex Token",
+    "decimals": 18,
+    "symbol": "CVX",
+    "logoURI": "../images/convex.svg"
+  },
+	{
```

Comment:
are these tabs instead of spaces?

Reply:
It's mixed tab and space, I'm going to add a linter + fix my local config

Reply:
No idea how I have a vim that believes tabs are acceptable, but here we are.